src/main/java/com/conveyal/datatools/manager/models/FeedVersionSummary.java

```diff
@@ -28,6 +37,13 @@ public class FeedVersionSummary extends Model implements Serializable {
     @JsonIgnore
     public ValidationResult validationResult;
     private PartialValidationSummary validationSummary;
+    public Date processedByExternalPublisher;
+    public Date sentToExternalPublisher;
+    public GtfsPlusValidation gtfsPlusValidation;
+    public String publishedVersionId;
+    public int publishedFeedVersionErrorCount;
+    public LocalDate publishedFeedVersionStartDate;
```

Comment:
Do we need to expose `publishedFeedVersionStartDate` and `publishedFeedVersionEndDate` in the summary? (They are not used in https://github.com/ibi-group/datatools-ui/pull/1044/ so far.